_posts/en/posts/2025-04-14-release-29.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Bitcoin Core 29.0 released
+name: blog-release-29.0
+id: en-blog-release-29.0
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 29.0 is now available.
+---
+Bitcoin Core version 29.0 is now available for [download][download
+page].  See the [release notes][release notes] for more information about the
+bug fixes in this release.
+
+With the release of this new major version, versions 26.x and older are at
+"Maintenance End" and will no longer receive updates. In accordance with the
+[security policy][security policy], two weeks after this release, medium and
+high severity vulnerabilities affecting versions 26.x (if any) will be
+disclosed.  Additionally, low severity vulnerabilities affecting versions 28.x
+(if any) will be disclosed.
```

Comment:
Should we say "in 2 weeks" or even give a specific date if you expect to publish today?

Reply:
Let's say 2 weeks. Sometimes we miss intended deadlines when we don't have people to review stuff.